src/screens/profile_details/components/connections/components/mobile/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classnames from 'classnames';
+import useTranslation from 'next-translate/useTranslation';
+import dayjs, { formatDayJs } from '@utils/dayjs';
+import { useRecoilValue } from 'recoil';
+import { readDate } from '@recoil/settings';
+import {
+  Divider,
+  Typography,
+} from '@material-ui/core';
+import { chainConfig } from '@src/configs';
+import Link from 'next/link';
+import { ACCOUNT_DETAILS } from '@utils/go_to_page';
+import { useStyles } from './styles';
+
+const Mobile: React.FC<{
+  className?: string;
+  items?: ProfileConnectionType[];
+}> = ({
+  className, items,
+}) => {
+  const dateFormat = useRecoilValue(readDate);
+  const classes = useStyles();
+  const { t } = useTranslation('accounts');
+
+  return (
+    <div className={classnames(className)}>
+      {items.map((x, i) => {
+        return (
+          <React.Fragment key={`votes-mobile-${i}`}>
+            <div className={classes.list}>
+              <div className={classes.item}>
+                <Typography variant="h4" className="label">
+                  {t('network')}
+                </Typography>
+                <Typography variant="body1" className="value">
+                  {x.network.toUpperCase()}
+                </Typography>
+              </div>
+              <div className={classes.item}>
+                <Typography variant="h4" className="label">
+                  {t('identifier')}
+                </Typography>
+                <Typography variant="body1" className="value">
+                  {new RegExp(`^(${chainConfig.prefix.account})`).test(x.identifier) && (
+                  <Link href={ACCOUNT_DETAILS(x.identifier)} passHref>
+                    <Typography variant="body1" className="value" component="a">
+                      {x.identifier}
+                    </Typography>
+                  </Link>
+                  )}
+                  {new RegExp(`^(${chainConfig.prefix.account})`).test(x.identifier) === false && x.identifier}
```

Comment:
Do this before return on line 29. It's bad practice to do logic testing here and makes the code slightly hard to read

Reply:
No problem.